### ISSUE TITLE
minio failing in CI

### ISSUE THREAD

**Issue by bacongobbler** [deis/minio#107]
From https://ci.deis.io/job/workflow-test-pr/2197/artifact/2197/logs/deis-minio-miiij.log/*view*/:

```
2016/06/06 17:34:48 starting Minio server
2016/06/06 17:34:48 starting health check server on 0.0.0.0:8082
minio server /home/minio/
error at command wait
2016/06/06 17:34:48 Minio server error (exit status 127)
```

perhaps this is fixed via https://github.com/deis/minio/pull/106?

**Comment by arschles** [comment]
I believe this is fixed in #106. I'll mark as such in that PR

**Comment by kmala** [comment]
yes..this is fixed by #106

**Comment by arschles** [comment]
Closed by #106
